apps/test/unit/code-studio/pd/application/TeacherApplicationIntegrationTest.js

```diff
@@ -55,13 +59,15 @@ describe('TeacherApplication', () => {
     sinon.assert.calledThrice(firehoseClient.putRecord);
   });
   it('Does not set schoolId if not provided', () => {
-    const page = mount(<TeacherApplication {...defaultProps} />);
+    const page = mount(<FindYourRegion {...defaultProps} />);
     expect(page.find('SchoolAutocompleteDropdown').prop('value')).to.equal(
       undefined
     );
   });
   it('Sets the school dropdown value from props', () => {
-    const page = mount(<TeacherApplication {...defaultProps} schoolId="50" />);
+    const page = mount(
```

Comment:
The SchoolAutocompleteDropdown element was on the first page of the original application (so having it mount <TeacherApplication> worked before to access the SchoolAutocompleteDropdown, but now that the SchoolAutocompleteDropdown is on the second page ("Find Your Region"), I changed how these tests are set up. I'm not sure how good this fix is, so I'm totally down to change this if anyone has any thoughts on how to better approach this!

Reply:
This makes sense to me, but I don't have a ton of context :)